src/main/kotlin/camp/nextstep/edu/racingcar/interfaces/CarTrackSnapshot.kt

```diff
@@ -0,0 +1,16 @@
+package camp.nextstep.edu.racingcar.interfaces
+
+import camp.nextstep.edu.racingcar.racing.Car
+import camp.nextstep.edu.racingcar.racing.CarRaceEvent
+import camp.nextstep.edu.racingcar.racing.CarRaceEvent.Companion.merge
+
+class CarTrackSnapshot(car: Car, events: List<CarRaceEvent>) {
+
+    private val carName = car.name
+    private val track = events.merge()
+    private val traces = track.traces().joinToString(" ") { if (it) "⎼" else "" }
```

Comment:
```suggestion
    private val traces = track.traces().filter { it }.joinToString(" ") { "⎼" }
```
Nit: 요런 방법도 있어 보이네요. if vs filter